### PR TITLE
[OVMM] Teleport action support, planner fixes, metric logging updates

### PR DESCRIPTION
**Pull request by yvsriram**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

File: projects/habitat_ovmm/eval_vectorized.py [hunk 1]
```diff
@@ -154,67 +160,62 @@ def stop():
...
-                                    for k1, v1 in aggregated_metrics.items()
-                                    for k2, v2 in {
-                                        f"{k1}/mean": np.round(np.mean(v1), 2),
-                                    }.items()
-                                }.items()
-                            )
-                        )
-
-                    if len(episode_metrics) % 5 == 0:
+                    if len(episode_metrics) % 1 == 0:
```
Comment (cpaxton) [inline projects/habitat_ovmm/eval_vectorized.py:218]: % 1 should maybe be a parameter somewhere? right now this is writing every time right

File: src/home_robot/home_robot/navigation_planner/discrete_planner.py [hunk 2]
```diff
@@ -287,7 +286,7 @@ def plan(
                 #     print("Nowhere left to explore. Stopping.")
```
Comment (cpaxton) [inline src/home_robot/home_robot/navigation_planner/discrete_planner.py:286]: should this still be commented out?

**Review by cpaxton — APPROVED** [review]
lgtm

File: src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py [hunk 3]
```diff
@@ -142,34 +156,34 @@ def _hardcoded_place(self):
...
         """State machine"""
-        vis_inputs = self._get_vis_inputs(obs)
+        info = self._get_info(obs)
         turn_angle = self.config.ENVIRONMENT.turn_angle
 
         self.timesteps[0] += 1
         if self.states[0] == Skill.NAV_TO_OBJ:
             if self.skip_nav_to_obj:
-                self._switch_to_next_skill(e=0)
+                info = self._switch_to_next_skill(e=0, info=info)
```
Comment (JHurricane96) [inline src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py:165]: It looks like nothing is being returned in any of the conditions when a skill is to be skipped, in which case we default to returning the stop action at the end of the `act` method. If I understood this correctly, then this is the wrong behavior and we should return the action and info in each of the conditions.

Comment (yvsriram) [reply src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py:165]: Yes, if  the current skill is to be skipped, we just move to the next skill in the same time step. STOP is only called if all skills skip.

Comment (yvsriram) [reply src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py:165]: So the current behvaior (of not returning and trying the next if condition) is intended.

File: src/home_robot/home_robot/navigation_planner/discrete_planner.py (hunk 2 above)
Comment (yvsriram) [reply src/home_robot/home_robot/navigation_planner/discrete_planner.py:286]: I think this need not be commented out. But I will remove this after testing in a future PR along with other changes (for error reporting).

**Pull request merged**